README.md

```diff
@@ -1 +1,57 @@
-# Ruby-project1
\ No newline at end of file
+# Capstone Project NTML & CSS
+
+This is the Ruby project #1, from the Microverse's main curriculum and aims to build two algorithms similar to the sorting method, without using those methods.
+
+![screenshot](./assets/Ruby-pro_1.png)
+
+## Built With
+
+- RUBY
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Live Server plugin for Visual Studio Code
+
+### Setup
+
+1- Clone the repository
+
+```
+https://github.com/realisticattorney/CapstoneProject.git
+```
+
+2- Open the folder on VS Code, right click the index.html file and click on "Open With Live Server".
+
+3- Everything should be running by now.
+
```

Comment:
- [ ] You do not have an HTML file in this project, please update these instructions

- [ ] **OPTIONAL** If you want your user to see a live demo of your code, you can use repl.it or gitpod.io. In gitpod.io all you have to do is append gitpod.io/ to the URL of your repo, like this: 
https://gitpod.io/github.com/Wusinho/Ruby-project1/tree/Feature-Buble_sort